.gitignore

```diff
@@ -5,6 +5,7 @@
 .hypothesis/
 .pytest_cache/
 .mypy_cache/
+.idea/
```

Comment:
I recommend to use a gitignore in your home directory for editor specifics. We can keep this, though, since pycharm, etc. is fairly common